server/events/comment_parser.go

```diff
@@ -186,11 +187,19 @@ func (e *CommentParser) Parse(comment string, vcsHost models.VCSHostType) Commen
 		flagSet.StringVarP(&dir, dirFlagLong, dirFlagShort, "", "Apply the plan for this directory, relative to root of repo, ex. 'child/dir'.")
 		flagSet.StringVarP(&project, projectFlagLong, projectFlagShort, "", fmt.Sprintf("Apply the plan for this project. Refers to the name of the project configured in %s. Cannot be used at same time as workspace or dir flags.", yaml.AtlantisYAMLFilename))
 		flagSet.BoolVarP(&verbose, verboseFlagLong, verboseFlagShort, false, "Append Atlantis log to comment.")
+	case models.ApprovePolicyCommand.String():
+		name = models.ApprovePolicyCommand
+		flagSet = pflag.NewFlagSet(models.ApprovePolicyCommand.String(), pflag.ContinueOnError)
+		flagSet.SetOutput(ioutil.Discard)
+		flagSet.StringVarP(&workspace, workspaceFlagLong, workspaceFlagShort, "", "Approve policies for this Terraform workspace.")
```

Comment:
why is this function called StringVarP?

Reply:
This is a copy paste from the commands above. For the first iteration I will remove all the extra flags, we can keep verbose and assume we always approve everything with 1 command.